static/src/javascripts/bootstraps/identity-common.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Identity crap that has to run on every page (putting in usernames, avatars, etc.)
+ *
+ * Explicitly NOT stuff that only runs on the identity pages. Put that in profile.js or I will hunt you down. I WILL
+ * HUNT YOU DOWN.
```

Comment:
:+1: reminds me of this toptier comment https://github.com/guardian/frontend/blob/master/common/app/model/meta.scala#L196-L210